RELEASE-NOTES.md

```diff
@@ -1,3 +1,28 @@
+# Google Cloud IoT Core Device SDK for Embedded C version  1.0.1
+#### March 29, 2019
```

Comment:
probably this should be updated to current date

Reply:
Yes, as I said, "I will update the release date upon review approval."